### ISSUE TITLE
Add Open Preview button 

### ISSUE THREAD

**Issue by smit1678** [hotosm/oam-uploader#75]
When uploading imagery, one disconnect is that there is a slight delay from when the image is finished processing and it is indexed into the catalog. This is because the catalog indexing runs every 5 mins and so the range in when the image shows up on map.openaerialmap.org is between 0 and 5 mins. In addition, there is a preview mode of the TMS available immediately. 

Idea here is to add a `Open preview` button to the `/status` page ([example](https://upload.openaerialmap.org/#/status/58fffb2490a38300103c37d0)) to improve getting access to the imagery immediately. 

Something along the lines of: 

![](https://api.monosnap.com/rpc/file/download?id=R4STqm4g6WZgaTKK9REyvDXH1jekoQ)

**Comment by tombh** [comment]
Just putting it out there, but what about just throwing a 5 minute `sleep()` in the uploader? Or better, poll the catalog API? It wouldn't be that hard.

**Comment by smit1678** [comment]
> Or better, poll the catalog API? It wouldn't be that hard.

Yeah, I'd be open to that solution as well. That's actually a better solution.

**Comment by cgiovando** [comment]
Was this solution actually implemented?

I also noticed that when the uploaded completes, it does not always appear on the `Latest Imagery` showcase in the [OAM homepage](http://openaerialmap.org/)
